src/css/globals/grid.css

```diff
@@ -17,6 +17,7 @@
 
 .flex {
   display: flex;
+  justify-content: center;
```

Comment:
if you add this property here, it will apply the property to all elements with the flex class, which is not desirable.

In line 50 of this file, there is another class named `justify-center` which adds the same property to any elements with that class. 

Therefore, you should take this property out of `.flex`, and then add the `justify-center` class to the .container.wrapper div in top-banner.hbs. That should give the `justify-content: center` property to the div without affecting `.flex`

Reply:
fixed